features/aanschrijfwijze.feature

```diff
@@ -66,12 +66,12 @@ Functionaliteit: Als gemeente wil ik de juiste en consistente aanschrijfwijze va
         - de adellijke titel en het predikaat worden met kleine letters opgenomen
         - de adellijke titel of het predikaat van de partner wordt niet opgenomen bij aanduiding naamgebruik "E" (eigen naam)
         - de adellijke titel of het predikaat van de persoon wordt niet opgenomen bij aanduiding naamgebruik "P" (partner naam)
-        - de adellijke titel van de partner wordt niet opgenomen als geslachtsaanduiding van de persoon ongelijk is aan "V" (vrouw) of als de geslachtsaanduiding van de partner ongelijk is aan "M" (man)
+        - de adellijke titel van de partner wordt niet opgenomen als geslachtsaanduiding van de persoon gelijk is aan "O" (onbekend)
```

Comment:
ik ben het niet eens met deze aanpassing. Wel zie ik dat de oude beschrijving een dubbele ontkenning heeft (niet...ongelijk), want niet al te duidelijk is.

De adellijke titel van de partner wordt alleen opgenomen als geslachtsaanduiding van de persoon gelijk is aan "V" (vrouw) en de geslachtsaanduiding van de partner gelijk is aan "M" (man).

Reply:
Ah, dat verandert veel.

Reply:
Aangepast